src/tiled/editabletilelayer.cpp

```diff
@@ -101,6 +106,51 @@ TileLayerEdit *EditableTileLayer::edit()
     return new TileLayerEdit(this);
 }
 
+TileLayerWangEdit *EditableTileLayer::wangEdit(EditableWangSet *wangSet)
+{
+    return new TileLayerWangEdit(this, wangSet);
+}
+
+void EditableTileLayer::applyChangesFrom(TileLayer *changes, bool mergeable) 
```

Comment:
This is moved from `tilelayeredit.cpp::apply`, because this code needs to run now for both `tilelayeredit` and `tilelayerwangedit`.